server/lib/routes/get-verify-email.js

```diff
@@ -116,3 +128,16 @@ module.exports = function () {
     }
   };
 };
+
+function getIpAddress(req) {
```

Comment:
This duplicates logic [found in route_logging.js](https://github.com/mozilla/fxa-content-server/blob/issue-5188-max-event-offset/server/lib/logging/route_logging.js#L24), is it possible to extract this into a common area to be shared?